.github/workflows/frontend.yml

```diff
@@ -14,7 +14,7 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-      - name: Fetch all history for all tags and branches
+      - name: fetch history, tags and branches
```

Comment:
@Rugvip as you can see, it was necessary to add a new step do download all the branches available, because by default the action only checks out the latest commit (**github.sha**) from the branch ref (**github.ref**) when the workflow is triggered. So no have other commits or branches be checked out for comparison (**git diff**).

If we want to compare different branches in the repository, we need to execute extra commands to checkout other branches, like this I've added.

Reply:
Hmm, this ends up downloading a lot of data though. It might be possible to first checkout master, and then the branch we're testing?

Reply:
Maybe, I can try this! :+1: